firmware/thruna/DCM/Src/App/states/App_DriveState.c

```diff
@@ -67,6 +69,13 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
     {
         App_SetPeriodicCanSignals_TorqueRequests();
```

Comment:
This function is going to send a torque request. Only this or App_Run_Regen() should run in a single cycle, not both.

Reply:
ok i'll look into it